app/konnect/gateway-manager/plugins/add-custom-plugin.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Managing custom plugins in Konnect
+content_type: how-to
+---
+
+You can manage custom plugins via the {{site.konnect_short_name}} UI or 
+the {{site.konnect_short_name}} Custom Plugins API.
+
+If you need to update a schema for a plugin that has already been uploaded
+to {{site.konnect_short_name}}, there are a few considerations based on the type 
+of update. 
+See [Editing or deleting a custom plugin's schema](/konnect/gateway-manager/plugins/update-custom-plugin/) 
+for more information.
+
+{:.important}
+> **Caution**: Carefully test the operation of any custom plugins before deploying
+them to production. Kong is not responsible for the operation or support of any 
+custom plugins, including any performance impacts on your {{site.konnect_short_name}}
+or {{site.base_gateway}} deployments. 
```

Comment:
```suggestion
{:.important}
> **note**: Deploying custom plugins to production without testing them first could have unintended consequences for your system.
```

In general, this warning at the top of a doc like this makes it really uninviting for me. Like, I shouldn't do this because there is a chance I break my whole system and the org is washing their hands of any liability.

Reply:
I think the "washing our hands of liability" is the idea - this was a request from legal to add.

Reply:
I can move the warning down in the topic, but it must be here, and we can't cut it down as much as you have in the suggestion.